### ISSUE TITLE
Add support for preferredContentMode

### ISSUE THREAD

**Issue by vascofg** [react-native-community/react-native-webview#1460]
Starting with iOS 13, iPad devices request desktop websites by default.
This can be overridden by setting `preferredContentMode` like so (in Swift):
```swift
@available(iOS 13.0, *)
func webView(_ webView: WKWebView, decidePolicyFor navigationAction: WKNavigationAction, preferences: WKWebpagePreferences, decisionHandler: @escaping (WKNavigationActionPolicy, WKWebpagePreferences) -> Void) {
    self.webView(webView, decidePolicyFor: navigationAction) {
        preferences.preferredContentMode = .mobile
        decisionHandler($0, preferences)
    }
}
```

I'd suggest implementing this in react-native-webview, by passing an optional parameters that sets the `preferredContentMode` preference.

**Comment by lubbo** [comment]
It would be nice to have an optional parameter to control the behaviour!

**Comment by jumoel** [comment]
Should be fixed by #1538.
